server/metrics.go

```diff
@@ -71,6 +71,23 @@ var (
 			Name:      "balancers_count",
 			Help:      "Counter of balancers.",
 		}, []string{"result"})
+
+	txnCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "pd",
+			Subsystem: "txn",
+			Name:      "txns_count",
+			Help:      "Counter of txns.",
+		}, []string{"type"})
```

Comment:
It is better to use result instead of type, for you only want to trace the result of txns.